src/tools/illink/Directory.Build.props

```diff
@@ -1,16 +1,22 @@
 <Project>
+  <Import Project="..\..\..\Directory.Build.props" />
   <PropertyGroup>
+    <RunAnalyzers Condition="'$(DotNetBuildFromSource)' != 'true'">true</RunAnalyzers>
     <IsReferenceAssembly Condition="'$(IsReferenceAssembly)' == '' and '$([System.IO.Path]::GetFileName($(MSBuildProjectDirectory)))' == 'ref'">true</IsReferenceAssembly>
     <DisableImplicitNamespaceImports_DotNet>true</DisableImplicitNamespaceImports_DotNet>
     <!-- The TFM for the product (linker, task,.. not analyzer, which is netstandard) -->
     <NetCoreAppToolCurrent>net7.0</NetCoreAppToolCurrent>
     <!-- The TFM for all the tests - intentionally different since tests may depend on new framework APIs in order to validate
          linker interaction with them, linker itself doesn't need the new framework typically. -->
     <NetCoreAppTestsCurrent>net7.0</NetCoreAppTestsCurrent>
+    <!-- Arcade uses by default xunit as test runner, illink uses nunit instead. 
+         We need to setup UseVSTestRunner to use the runner VS is using (in this case nunit) -->
+    <UseVSTestRunner>true</UseVSTestRunner>
+    <!-- No symbols are produced for ref assemblies, but some parts of the SDK still expect pdbs, so we explicitly tell it there are none. -->
+    <!-- Must be set after importing Arcade to override its defaults. -->
+    <DebugType Condition=" '$(IsReferenceAssembly)' == 'true' ">none</DebugType>
```

Comment:
nit: Move this into the below PropertyGroup and remove the Condition from DebugType.